marklogic-data-hub/src/main/resources/hub-internal-artifacts/step-definitions/mastering/marklogic/matching-mastering.step.json

```diff
@@ -0,0 +1,20 @@
+{
+  "lang": "zxx",
+  "name": "matching-mastering",
+  "type": "mastering",
```

Comment:
We have the format "default-*" for all the OOTB step definitions . Can the step def names be "default-matching" and "default-merging" ?

Reply:
Changed to reflect the "default-*" convention with the latest commit.